### PR TITLE
Fixes #31 Creating the onboarding readme.

### LINKED ISSUE #31: create README.md in Registry posts area
There is now a link to Registry in onboarding (thanks to @englehorn 😄):
https://github.com/CodeForFoco/codeforfoco.github.io/tree/master/_posts/registry

A `README.md` there would also be helpful. GitHub website renders a `README.md` automatically if it finds one.

The `README.md` would include
 -  instructions on how to use `template.md` (very useful! thanks @itorain ! 😸 )
 - fork process and Pull Request process (this might already be covered elsewhere? a link from README itself to that would be directly useful too)

### PR DESCRIPTION
**Pull request by itorain**

### CODE REVIEW

File: _posts/registry/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+### How to create a new post
+The registry post use something called markdown to display. Markdown is a simple way to format plain text. You can learn more about it [here](https://daringfireball.net/projects/markdown/). Also you will need to know a bit about Github. You can learn more about Github [here](https://guides.github.com/). To complete the onboarding challenge follow the steps below. And remember if you need help, ask! Known of us here bite.
+1. Fork the repository
+1. In your local repository, copy the md template below
+1. Create a new file, and paste in the template.
+1. Edit the template with your information and message
+1. Save the file with the name in the form: YYYY-MM-DD-YOURTITLE.md
+1. Commit your changes
+1. Make a pull request to merge your post into master.
+
```
Comment (idcrook) [inline _posts/registry/README.md]: The markdown section below looks great. A header section title here would also provide useful visual identification

**Comment by idcrook** [comment]
This should cover #31.  Merging.

**Comment by idcrook** [comment]
thanks for this @itorain 👍

**Comment by itorain** [comment]
@idcrook No problem!

**Pull request merged**